src/Error/Middleware/ErrorHandlerMiddleware.php

```diff
@@ -139,7 +140,17 @@ protected function getRenderer($exception)
         if (is_string($this->exceptionRenderer)) {
             $class = App::className($this->exceptionRenderer, 'Error');
             if (!$class) {
-                throw new Exception("The '{$this->exceptionRenderer}' renderer class could not be found.");
+                throw new Exception(sprintf(
+                    "The '%s' renderer class could not be found.",
+                    $this->exceptionRenderer
+                ));
+            }
+            if (!$class instanceof ExceptionRendererInterface) {
```

Comment:
Technically you are right.
How many upgraded to Middleware yet however since 3.3 is out?
The benefit of this crucial missing piece of software contract that has been forgotten in the initial implementation might justify the migration note here.

I will go with the majority vote here.